Assets/Scripts/Entity/Enemy/MovementBehaviors/WanderMovement.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+
+public class WanderMovement : MonoBehaviour, IOnDeathController
```

Comment:
Did you see the already existing [WanderEnemyMovement](https://github.com/cap4053-cheeky-pixels/EmbodyGame/blob/2a2d36f0744969c09292d62cb9929acde2ec17a2/Assets/Scripts/Entity/Enemy/MovementBehaviors/WanderEnemyMovement.cs). Just saying because these two are named really similarly and may get confusing later.

Reply:
Yes, but I wanted to start fresh